CPDShell/Core/algorithms/BayesianCPD/abstracts/idetector.py

```diff
@@ -0,0 +1,45 @@
+"""
+MIT License
+
+Copyright (c) 2024 Alexey Tatyanenko
```

Comment:
Vladimir told us about some tool that automatically generates a header for each file. If anything, he's not like that...

Reply:
Yep for now I made it manually (also skipping empty inits), however this part is discussable. Which tool would be better to use?

Reply:
Look at https://github.com/vkutuev/cpd-datasets/blob/main/cpd_dataset/main.py#L7